lib/models/statistics_model/statistic/statistic.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'statistic.freezed.dart';
+part 'statistic.g.dart';
+
+@freezed
+class Statistic with _$Statistic {
+  const factory Statistic({
+    String? type,
+    dynamic value,
```

Comment:
Why the value variable is dynamic?

Reply:
it can be either int, string or null depending what stat is provided from Api, e.g. ball possession is represented as string, shots or goals are integers but most of stats can be null sometimes if no such event occurred during the game